static/app/views/starfish/queries/useFullSpanFromTrace.tsx

```diff
@@ -20,7 +22,18 @@ export function useFullSpanFromTrace(
     filters[SpanIndexedField.SPAN_GROUP] = group;
   }
 
-  const indexedSpansResponse = useIndexedSpans(filters, sorts, 1, enabled);
+  // TODO: we're using all SpanIndexedFields here, but maybe we should only use what we need?
+  // Truncate to 20 fields otherwise discover will complain.
+  const fields = Object.values(SpanIndexedField).slice(0, 20);
```

Comment:
Added a todo note here. We need to update this because `SpanIndexedField` is eventually going to grow beyond 20 elements and Discover doesn't support querying more than 20.

Reply:
Up to you, but IMO it's fine to let Discover complain, rather than truncating the fields, since that's a silent failure! Anyone who doesn't know about the limit would be really confused why fields they're asking for aren't coming back, and everyone should provide the list of fields they want, rather than defaulting to all fields coming back